app/controllers/api/v1/books_controller.rb

```diff
@@ -5,12 +5,23 @@ def index
     render json: books
   end
 
+  def csv
+    books = Book.find(params[:books])
+    puts Book.to_csv(books)
+    send_data Book.to_csv(books),
+                type:         "text/csv; charset=iso-8859-1; header=present",
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Unnecessary spacing detected.